MdeModulePkg/Library/MemoryBucketLib/MemoryBucketLib.inf

```diff
@@ -0,0 +1,36 @@
+## @file
+# Library the defines Runtime memory buckets for use during PEI
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = MemoryBucketLib

```

Comment:
Please prefix the library instance with the phase type, e.g. `PeiMemoryBucketLib`.

Reply:
This also affects the filenames.